core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -98,4 +107,25 @@ private function supports(AbstractDbafsMetadataEvent $event): bool
     {
         return 'tl_files' === $event->getTable();
     }
+
+    private function getDefaultLocales(): array
+    {
+        if (null !== $this->defaultLocales) {
+            return $this->defaultLocales;
+        }
+
+        $page = $GLOBALS['objPage'] ?? null;
```

Comment:
I also think we should use https://github.com/contao/contao/pull/3930 now :)

Reply:
see fb80c6d653017a787dd64119d6e3d4a86451a35a

Reply:
PHP 8 ❤️

Reply:
So much easier :smile: